### PR TITLE
Disable selected Tools menu items in dashboard

### PR DESCRIPTION
**Pull request by RussKie**
...as those only apply to a git repo.

## Merge strategy

I agree that the maintainer squash merge this PR (if the commit message is clear).

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

File: src/app/GitUI/CommandsDialogs/Menus/ToolsToolStripMenuItem.cs [hunk 1]
```diff
@@ -39,6 +39,8 @@ public override void RefreshShortcutKeys(IEnumerable<HotkeyCommand>? hotkeys)
         public override void RefreshState(bool bareRepository)
         {
             gitGUIToolStripMenuItem.Enabled = !bareRepository;
+            kGitToolStripMenuItem.Enabled = !bareRepository;
```
Comment (mstv) [inline src/app/GitUI/CommandsDialogs/Menus/ToolsToolStripMenuItem.cs:42]: GitK works well for bare repos, and it doesn't crash if there is no repo.

image

Would it really be worth passing this information to `RefreshState(bool bareRepository)`?

File: src/app/GitUI/CommandsDialogs/Menus/ToolsToolStripMenuItem.cs [hunk 2]
```diff
@@ -39,6 +39,8 @@ public override void RefreshShortcutKeys(IEnumerable<HotkeyCommand>? hotkeys)
...
+            gitBashToolStripMenuItem.Enabled = !bareRepository;
```
Comment (mstv) [inline src/app/GitUI/CommandsDialogs/Menus/ToolsToolStripMenuItem.cs:43]: A bash can be opened everywhere.
```suggestion
```

**Review by mstv — COMMENTED** [review]
🤔

**Review by gerhardol — COMMENTED** [review]
Agree with mstv, does not seem to be needed.
Especially git bash should be available, gitk does not matter.
The Copilot suggestion makes sense...

**Pull request closed without merging**